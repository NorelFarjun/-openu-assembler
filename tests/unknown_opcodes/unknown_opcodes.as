stoppp
STOP
ADD r1, r3
data HELLO
HELLO: CLR K
