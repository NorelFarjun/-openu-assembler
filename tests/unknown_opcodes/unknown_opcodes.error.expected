0: unknown instruction 'stoppp'
1: unknown instruction 'STOP'
2: unknown instruction 'ADD'
3: unknown instruction 'data'
4: unknown instruction 'CLR'
Bad input file - not outputting
